cmd/kube-apiserver/app/server.go

```diff
@@ -218,6 +218,7 @@ func CreateKubeAPIServerConfig(opts options.CompletedOptions) (
 	genericConfig, versionedInformers, storageFactory, err := controlplaneapiserver.BuildGenericConfig(
 		opts.CompletedOptions,
 		[]*runtime.Scheme{legacyscheme.Scheme, extensionsapiserver.Scheme, aggregatorscheme.Scheme},
+		controlplane.DefaultAPIResourceConfigSource(),
```

Comment:
note for reviewer: this was locally called from `BuildGenericConfig`, but we try to cut dependencies from `pkg/controlplane/apiserver` to `pkg/controlplane`. Hence, we move `DefaultAPIResourceConfigSource` and made it a parameter for `BuildGenericConfig` 🎉